### DISCUSSION TITLE
How to set up dev environment for iOS?

### DISCUSSION THREAD

**Discussion by mokpi** [cashapp/redwood#987] [Q&A] [answered]
I have no prior experience with iOS development.

When I open the project (specifically, `EmojiSearchApp.xcworkspace`) with XCode, I see that XCode have issues with finding references to the generated definitions. (Running the app with simulator works fine though)

<img width="500" alt="Screenshot 2023-04-28 at 1 05 55" src="https://user-images.githubusercontent.com/34705943/234921152-fa79eb65-5d05-40f9-b8d5-8c9745f790c7.png">

Since I want to make some UI changes to the sample project, I need to make changes on XCode also and I'd really love the help from XCode. How can I resolve this?

## Standard iOS development setup?

Also, is there any extra iOS development set up recommendation more than just "open xcworkspace with XCode" that I'm not aware of?

**Comment by JakeWharton** [comment, marked as answer]
I see that when I first open the project, but then after doing a build the types resolve properly.

<img width="1767" alt="Screen Shot 2023-04-27 at 12 28 06 PM" src="https://user-images.githubusercontent.com/66577/234928645-a0c0ab82-17fc-4c3c-9722-36d36bad7ef6.png">

**Reply by mokpi** [reply]
Thanks! It works just as you described and I can even follow the reference to see the generated source!
